Makefile

```diff
@@ -330,6 +330,33 @@ importer-image: PLATFORMS=linux/amd64
 importer-image: PUSH=--load
 importer-image: importer-image-build
 
+
+# Build the kueue-viz dashboard images (frontend and backend)
+.PHONY: kueue-viz-image-build
+kueue-viz-image-build:
+	@for component in backend frontend; do \
+		$(IMAGE_BUILD_CMD) \
+			-t $(IMAGE_REGISTRY)/kueue-viz-$$component:$(GIT_TAG) \
+			-t $(IMAGE_REGISTRY)/kueue-viz-$$component:$(RELEASE_BRANCH)-latest \
+			--platform=$(PLATFORMS) \
+			--build-arg BASE_IMAGE=$(BASE_IMAGE) \
+			--build-arg BUILDER_IMAGE=$(BUILDER_IMAGE) \
+			--build-arg CGO_ENABLED=$(CGO_ENABLED) \
+			$(PUSH) \
+			-f ./cmd/experimental/kueue-viz/$$component/Dockerfile ./cmd/experimental/kueue-viz/$$component; \
+	done
+
+.PHONY: kueue-viz-image-push
+kueue-viz-image-push: PUSH=--push
+kueue-viz-image-push: kueue-viz-image-build
+
+# Build a docker local us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue-viz image
+.PHONY: kueue-viz-image
+kueue-viz-image: PLATFORMS=linux/amd64,linux/arm64,linux/s390x,linux/ppc64le,darwin/amd64,darwin/arm64
```

Comment:
Actually, for consistency, I would suggest to declare the set of platforms as we do for `CLI_PLATFORMS`, say `UI_PLATFORMS`, or `VIZ_PLATFORMS` and map them to `PLATFORMS`.